packages/docs/src/pages/getting-started/gatsby.mdx

```diff
@@ -7,7 +7,7 @@ title: Getting Started with Gatsby
 To use Theme UI with [Gatsby][], install and use [`gatsby-plugin-theme-ui`](/packages/gatsby-plugin).
 
 ```sh
-npm i theme-ui gatsby-plugin-theme-ui
+npm i theme-ui gatsby-plugin-theme-ui @emotion/react @emotion/styled @mdx-js/react@v1
```

Comment:
`gatsby-plugin-mdx` doesn't support ESM-only MDX v2 yet, so people need to install v1